tests/config.yml

```diff
@@ -1,3 +1,3 @@
 ---
 modules:
-  python_requires: ">=3.8"
+  python_requires: ">=3.9"
```

Comment:
If we're _not_ incrementing to 3.8 and higher, this change is also unnecessary.

Reply:
Never mind, it appears that we do indeed need to [support a minimum Python version of 3.9 past April 2025](https://aws.amazon.com/blogs/developer/python-support-policy-updates-for-aws-sdks-and-tools/).